src/build/tenscript/mod.rs

```diff
@@ -1,5 +1,5 @@
-use std::collections::HashMap;
 use std::fmt::{Display, Formatter};
+use hashbrown::HashMap;
```

Comment:
So goofy that this is necessary.

Reply:
Goofy is not the word!  We need to get the word out.